src/file/file.service.spec.ts

```diff
@@ -2,13 +2,13 @@ import * as chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
```

Comment:
Do all the tests run?

Reply:
it runs but fails because of the promiseAllSettled. might be because of the node version

Reply:
its not because node, the typescript version was 3.4.5 and it didnt work, so from its working on version 3.8

Reply:
works perfectly fine now, all done.